### PR TITLE
added landing page

### PR DESCRIPTION
**Pull request by Samarth-5109**
The landing page of my e-commerce website"Tyre Club"
Live Link: https://vigorous-leakey-e8e065.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,107 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <!-- <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up/2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/> -->
```
Comment (Kothalkarpriya) [inline index.html:8]: remove these comments if they are not useful

File: index.html [hunk 2]
```diff
@@ -4,9 +4,107 @@
...
+            <span class="brand_name">Tyre Club</span>
+        </div>
+        <nav class="navbar">
+            <ul class="nav-items">
+                <!-- <li><a href="">Home</a></li> -->
+                <li class="list-items "><i class="far fa-location"></i><a href="#" class="doc"> Select Your Location</a></li>
+                <li><a href="">Login</a></li>
+                <li><a href="">Sign-Up</a></li>
+                <div class="badges-on-icon-container">
+                    <li class="list-items item-2"><a href=""><i class="fas fa-shopping-cart icon"></i>
```
Comment (Kothalkarpriya) [inline index.html:25]: use dead links for anchor tags

File: index.html [hunk 3]
```diff
@@ -4,9 +4,107 @@
...
+            </ul>
+        </nav>
+    </header>
+    <div>
+        <section class="products">
+            <nav class="products-nav">
+                <ul class="products-list">
+                    <li><a class="product" href="">Car Tyres</a></li>
+                    <li><a class="product" href="">Bike Tyres</a></li>
+                    <li><a class="product" href="">Lubricants</a></li>
```
Comment (Kothalkarpriya) [inline index.html]: here also use # for links

File: index.html [hunk 4]
```diff
@@ -4,9 +4,107 @@
...
+            <div class="brand-names">
+                <!-- <img class="brand-logos" src="./assets/images/mrf-logo.png" alt=""> -->
+                <img class="brand-logos" src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/ba/CEAT_Logo.svg/1200px-CEAT_Logo.svg.png" alt="">
+                <img class="brand-logos" src="./assets/images/continental-logo.png" alt="">
+                <img class="brand-logos" src="https://www.carlogos.org/tire-logos/bridgestone-logo-5500x1200.png" alt="">
+                <img class="brand-logos" src="./assets/images/mrf-logo.png" alt="">
+                <img class="brand-logos" src="https://www.pngkey.com/png/detail/484-4848018_apollo-tyres-logo-png.png" alt="">
+                <img class="brand-logos" src="https://www.carlogos.org/tire-logos/michelin-logo-3600x700.png" alt="">
+                <img class="brand-logos" src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d9/Logo_Pirelli.svg/2560px-Logo_Pirelli.svg.png" alt="">
+
```
Comment (Kothalkarpriya) [inline index.html]: remove white space

File: index.html [hunk 5]
```diff
@@ -4,9 +4,107 @@
...
+            <div class="details">
+                <p class="main-title title-brand">Search Car Tyres By Brands</p>
+                <p class="about-brand">Choose from a wide range of car tyre brands and buy them online at best prices. Our customer friendly fitment partners across India would assist you with world-class service</p>
+            </div>
+            <div class="brand-names">
+                <!-- <img class="brand-logos" src="./assets/images/mrf-logo.png" alt=""> -->
+                <img class="brand-logos" src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/ba/CEAT_Logo.svg/1200px-CEAT_Logo.svg.png" alt="">
+                <img class="brand-logos" src="./assets/images/continental-logo.png" alt="">
+                <img class="brand-logos" src="https://www.carlogos.org/tire-logos/bridgestone-logo-5500x1200.png" alt="">
+                <img class="brand-logos" src="./assets/images/mrf-logo.png" alt="">
```
Comment (Kothalkarpriya) [inline index.html:62]: for images do use / before closing it. As it is Self closing tag

File: index.html [hunk 6]
```diff
@@ -4,9 +4,107 @@
...
+                <h3 class="works-heading">get door-step delivery</h3>
+                <p>Get your tyres delivered at your door-step usually in 3 business days*</p>
+            </div>
+        </section>
+
+        <section class="footer">
+            <footer class="footer-container">
+                <div class="social-media-links">
+                <a href=""><i class="fab fa-twitter  sm-links ecom"></i></a>
+                <a href=""><i class="fab fa-github sm-links ecom"></i></a>
```
Comment (Kothalkarpriya) [inline index.html]: here also use #

File: styles/landing.css [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+    height: 60vh;
+    /* background-color: var(--primary-color); */
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+.details{
```
Comment (Kothalkarpriya) [inline styles/landing.css]: remove if not needed blank selector

File: styles/main.css [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+.branding{
+    display: flex;
+    align-items: center;
+    margin-left: 2rem;
+    padding: 1rem;
+    
+}
+
+/* .brand_image{
+    width: 3.5rem;
```
Comment (Kothalkarpriya) [inline styles/main.css:42]: remove if not needed

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Look into my comments

File: styles/main.css [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+    height: 1rem;
+    text-align: center;
+    border-radius: 50%;
+    position: absolute;
+    color: var(--secondary-color);
+
+}
+
+.nfn-cart{
+    /* left:11rem; */
```
Comment (dev-enforced) [inline styles/main.css]: Remove this comment if not needed

File: styles/main.css [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+a{
+    text-decoration: none;
+    color: var(--primary-color);
+    margin-right:1rem;
+}
+
+.nav-container{
+    display: flex;
+    align-items: center;
+    /* background-color: var(--secondary-color); */
```
Comment (dev-enforced) [inline styles/main.css]: Remove this one too

File: styles/landing.css [hunk 11]
```diff
@@ -0,0 +1,109 @@
+@import url("./main.css");
+
+.products-nav {
+  display: flex;
+  justify-content: space-around;
+}
+
+.product {
+  font-weight: bold;
```
Comment (dev-enforced) [inline styles/landing.css:9]: This font-weight has been used multiple times so you can create a utility class for it named as txt-bold and use it wherever needed in the HTML file

File: styles/landing.css [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.about-brand {
+  width: 55rem;
+}
+
+.brand-names {
```
Comment (dev-enforced) [inline styles/landing.css:65]: You can create a utility class for it named as .flex-full-center{ //properties here} and use it wherever needed

File: styles/landing.css [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+.intro {
+  color: var(--secondary-color);
+  padding-top: 2rem;
+  padding-left: 6rem;
+}
+
+.main-title {
+  margin-bottom: 2rem;
+  font-size: 1.8rem;
+  text-transform: uppercase;
```
Comment (dev-enforced) [inline styles/landing.css:35]: This line is also used multiple time so you can create a utility class as txt-uppercase and use it in the HTML

**Review by dev-enforced — APPROVED** [review]
The code looks good to me . Suggested some minor changes which shall be done after you merge

**Review by Kothalkarpriya — APPROVED** [review]

**Pull request merged**